arrow/src/main/java/org/apache/iceberg/arrow/vectorized/VectorHolder.java

```diff
@@ -95,8 +111,12 @@ public NullabilityHolder nullabilityHolder() {
     return nullabilityHolder;
   }
 
-  public Type icebergType() {
```

Comment:
this incurs a change of public method, which might not be preferred. Instead of using 

```
private final Type physicalType;
  private final Type logicalType;
```

can we use

```
private final Type icebergType;
private final Type logicalType;
```

which also matches later changes better.

Reply:
Reverted the unnecessary change.